include/swupd-build-variant.h

```diff
@@ -25,6 +34,7 @@
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions " TAR_XATTR_ARGS
 #else /* SWUPD_WITHOUT_XATTRS */
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions "
-#endif
+
+#endif /* SWUPD_WITH_BSDTAR */
```

Comment:
Sorry, I should have been more specific... The comment should be on line 31, correct?